test/screens/v2/widget_instance/evergreen_content_test.exs

```diff
@@ -44,8 +57,12 @@ defmodule Screens.V2.WidgetInstance.EvergreenContentTest do
   end
 
   describe "valid_candidate?/1" do
-    test "returns true", %{widget: widget} do
+    test "returns true with blank schedule", %{widget: widget} do
       assert WidgetInstance.valid_candidate?(widget)
     end
+
+    test "returns false with old schedule", %{widget_old_schedule: widget_old_schedule} do
+      assert !WidgetInstance.valid_candidate?(widget_old_schedule)
```

Comment:
This is fine, but just FYI there's also a [`refute`](https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#refute/1) assertion available for things like this.